site/en/tutorials/distribute/dtensor_keras_tutorial.ipynb

```diff
@@ -99,7 +102,7 @@
       },
       "outputs": [],
       "source": [
-        "!pip install --quiet --upgrade --pre tensorflow tensorflow-datasets"
+        "!pip install --quiet --upgrade tensorflow tensorflow-datasets"
```

Comment:
Same.

Reply:
Will remove this, and lint the notebooks